### PR TITLE
Equativ: SmartAdserver alias with update to use mtype

### PR DESCRIPTION
**Pull request by rneuplanche**
In order to support correctly multi impression type bid request, we need to change the way we compute the BidType in the Equativ adapter

### CODE REVIEW

**Review by mdaddario-STR — APPROVED** [review]
Looks good. Does this take care of prebid server v2 and v3?

**Comment by rneuplanche** [comment]
> Looks good. Does this take care of prebid server v2 and v3?
@mdaddario-STR 

Not sure if the Prebid team is still publishing updates on PBS V2 🤔

**Comment by rneuplanche** [comment]
Hello @przemkaczmarek could we have a review on this ? I don't know the process on your end ?
Thanks in advance 🙏

**Comment by rneuplanche** [comment]
Hello @przemkaczmarek or @bsardo could we have a review on this ? I don't know the process on your end ?
Thanks in advance 🙏

File: adapters/smartadserver/smartadserver_test.go
Comment (przemkaczmarek) [inline adapters/smartadserver/smartadserver_test.go:46]: Why did You add those test? 
correctly is to test this through jsons

Comment (rneuplanche) [reply adapters/smartadserver/smartadserver_test.go]: I removed it. It's indeed already tested by jsons

**Review by przemkaczmarek — APPROVED** [review]

**Review by przemkaczmarek — DISMISSED** [review]

**Comment by rneuplanche** [comment]
Hello @SyntaxNode could we have a second review on this please 🙏

**Comment by rneuplanche** [comment]
Hello @SyntaxNode or @bsardo is it possible to have a second review on this please 🙏

**Review by bsardo — COMMENTED** [review]
Hi @rneuplanche, is this a rebranding? I see that you're adding Equativ as an alias of smartadserver. Are you planning to transition your customers from smartadserver to equativ including asking them to use the equativ bidder code? I see that the support email address for smartadserver has been an equativ domain email address for over a year so it appears there has been a relationship here for quite some time.

File: adapters/smartadserver/smartadservertest/exemplary/multi-impression-types-video-banner.json
Comment (rneuplanche) [reply adapters/smartadserver/smartadservertest/exemplary/multi-impression-types-video-banner.json:1]: You're right, I changed it for a native/audio test as audio was not well covered

File: adapters/smartadserver/smartadservertest/exemplary/multi-banner-multi-imp-type.json
Comment (rneuplanche) [reply adapters/smartadserver/smartadservertest/exemplary/multi-banner-multi-imp-type.json:1]: Renamed 👍

File: static/bidder-info/smartadserver.yaml
Comment (rneuplanche) [reply static/bidder-info/smartadserver.yaml]: Is there a way to mention that we support multi format in the doc ? And is there an official documentation on supported fields ? I couldn't found it

**Comment by rneuplanche** [comment]
Hello @bsardo, it has been nearly two years since [we rebranded from Smartadserver to Equativ](https://equativ.com/blog/press-release/smart-adserver-rebrands-as-equativ/). While our company remains unchanged, we are actively working to phase out references to our previous name. As part of this effort, we have introduced Equativ as an alias for Smartadserver, marking the first step in a gradual migration process aimed at aligning all of our branding and communications with our new identity.

File: static/bidder-info/smartadserver.yaml
Comment (bsardo) [reply static/bidder-info/smartadserver.yaml]: You can declare multi format support in the docs. See how other bidders have done it: https://github.com/search?q=repo%3Aprebid%2Fprebid.github.io%20multiformat_supported&type=code

**Review by bsardo — APPROVED** [review]

**Comment by rneuplanche** [comment]
Thanks for your approvals @przemkaczmarek & @bsardo 🙏 
I'm not allowed to merge anyway. Could you please merge it for me ?

**Pull request merged**
